algorithms-and-data-structures/basic-javascript/index110.js

```diff
@@ -0,0 +1,11 @@
+let xp = 0;
+let health = 100;
+let gold = 50;
+let currentWeapon = 0;
+let fighting;
+let monsterHealth;
+let inventory = ["stick"];
+
+let button1 = document.querySelector("#button1");
+
+// You can also declare variables with the `const` key word. Since `button1` is a constant that will never change, switch the `let` keyword that declares the variable to `const`.
```

Comment:
```suggestion
/*
You can also declare variables with the `const` key word.
Since `button1` is a constant that will never change, switch the `let` keyword to `const`.
*/
```